ts/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "grpc-web",
+  "version": "0.1.2",
+  "description": "gRPC browser requests (in TypeScript)",
```

Comment:
Also is there anythign preventing this being usable from JS?

Can you add a section?